config.toml

```diff
@@ -1,13 +1,13 @@
-[db]
-  DataDir = "/home/d/.gossamer"
+[global]
+data-dir = "/home/d/.gossamer"
```

Comment:
i don't think i have a /home/d :/

Reply:
Changed to `./.gossamer`

Reply:
and you can really change this to whatever your preference is @noot